File: Easy/C/AgeDistribution.c
#include <stdio.h>
#include <string.h>

void getYear(char line[]);

int main(int argc, const char* argv[]){
	FILE *file = fopen(argv[1], "r");
	char line[1024];
	while(fgets(line, 1024, file)){
			getYear(line);
			printf("\n");
	}
	return 0;
}


void getYear(char *line){
		int test = atoi(line);
		
		if(test >= 0  && test <= 2) 
			printf("Still in Mama's arms");
		else if(test >= 3 && test <= 4) 
			printf("Preschool Maniac");
		else if(test >= 5 && test <= 11) 
			printf("Elementary  school");
		else if(test >= 12 && test <= 14) 
			printf("Middle school");
		else if(test >= 15 && test <= 18) 
			printf("High school");
		else if(test >= 19 && test <= 22) 
			printf("College");
		else if(test >= 23 && test <= 65) 
			printf("Working for the man");
		else if(test >= 66 && test <= 100)
			printf("The Golden Years");
		else
			printf("This program is for humans");
	}

File: Easy/JAVA/AgeDistribution.java
import java.io.File;
import java.io.FileReader;
import java.io.BufferedReader;
import java.io.IOException;
public class AgeDistribution{
	public static void main(String[] args)throws IOException{
		File file = new File(args[0]);
		BufferedReader buffer = new BufferedReader(new FileReader(file));
		String line;
		
		while((line = buffer.readLine()) != null){
				line = line.trim();
				getYear(line);
		}//END WHILE
	}//END MAIN
	static void getYear(String line)
	{
			int test = Integer.valueOf(line);
			
			if(test < 0 || test > 100)
				System.out.println("This program is for humans");
			if(test <= 2) 
				System.out.println("Still in Mama's arms");
			else if(test <= 4) 
				System.out.println("Preschool Maniac");
			else if(test <= 11) 
				System.out.println("Elementary  school");
			else if(test <= 14) 
				System.out.println("Middle school");
			else if(test <= 18) 
				System.out.println("High school");
			else if(test <= 22) 
				System.out.println("College");
			else if(test <= 65) 
				System.out.println("Working for the man");
			else  
				System.out.println("The Golden Years");

	}
}//END CLASS
